spec/unit/shoryuken/error_handler_spec.rb

```diff
@@ -0,0 +1,53 @@
+require 'spec_helper'
+require 'airbrake/shoryuken/error_handler'
```

Comment:
@kyrylo it is, beause [it's not requiring the error handler unless Shoryuken defined](https://github.com/phstc/airbrake/blob/41dc8f28970694621764206edf26e1d31fbd1563/lib/airbrake.rb#L26). And to have Shoryuken defined, users need to explicit add Shoryuken in their Gemfiles.